cli/src/commands/git/push.rs

```diff
@@ -393,6 +393,27 @@ pub fn cmd_git_push(
             ),
             remote = remote.as_symbol()
         );
+
+        if bookmark_updates.is_empty() {
+            let deleted_bookmarks = tx
+                .repo()
+                .view()
+                .local_remote_bookmarks(remote)
+                .filter(|(_, targets)| {
+                    !targets.local_target.is_present()
+                        && matches!(classify_bookmark_push_action(*targets),
+                        BookmarkPushAction::Update(update) if update.new_target.is_none())
+                })
+                .count();
+
+            if deleted_bookmarks > 0 {
+                writeln!(
+                    ui.hint_default(),
+                    "You can push {deleted_bookmarks} deleted bookmark{s} with `--deleted`",
```

Comment:
nit: we avoid ad-hoc singular/plural handling. just use `N bookmarks` or `N bookmark(s)`.